Graphics/GraphicsEngineD3DBase/include/EngineFactoryD3DBase.hpp

```diff
@@ -110,12 +109,10 @@ class EngineFactoryD3DBase : public EngineFactoryBase<BaseInterface>
 
         UINT numModes = 0;
         // Get the number of elements
-        auto hr = pOutput->GetDisplayModeList(DXIGFormat, 0, &numModes, NULL);
```

Comment:
Why was this removed?

Reply:
It seemed to be unused and was causing an unused variable warning. I thought it was dead leftover code so I removed it. Can put it back.

Reply:
The function `GetDisplayModeList` does what the name suggests - it returns the list of display modes through output parameters.
Removing the function will break display mode enumeration.

Reply:
I see now, sorry about that will put it back.